src/data/finsupp/basic.lean

```diff
@@ -1470,6 +1470,43 @@ end
 
 end comap_domain
 
+/-! ### Declarations about `equiv_congr_left` -/
+
+section equiv_congr_left
+
+variable [has_zero M]
+
+/-- Given `f : α ≃ β`, we can map `l : α →₀ M` to  `equiv_map_domain f l : β →₀ M` (computably)
+by mapping the support forwards and the function backwards. -/
+def equiv_map_domain (f : α ≃ β) (l : α →₀ M) : β →₀ M :=
```

Comment:
Isn't this `finsupp.dom_congr`?

Reply:
Yes, except it's computable and has a better defeq for function application.

Reply:
Should we redefine dom_congr this way instead then?